package.json

```diff
@@ -21,16 +21,17 @@
     "test:e2e": "jest --config ./test/jest-e2e.json --runInBand",
     "test:e2e:cov": "jest --coverage --config ./test/jest-e2e.json --runInBand",
     "test:all": "jest --config ./test/jest-all.json --runInBand",
-    "test:all:cov": "jest --coverage --config ./test/jest-all.json --runInBand",
-    "postinstall": "husky install"
+    "test:all:cov": "jest --coverage --config ./test/jest-all.json --runInBand"
   },
   "dependencies": {
+    "@nestjs/cli": "^10.0.5",
     "@nestjs/common": "^10.0.3",
     "@nestjs/config": "^3.0.0",
     "@nestjs/core": "^10.0.3",
     "@nestjs/platform-express": "^10.0.3",
     "@nestjs/swagger": "^7.0.7",
     "@node-redis/json": "^1.0.2",
+    "@types/lodash": "^4.14.195",
```

Comment:
After some research, I finally found that not having this concrete dependency on the production image building process, makes the compilation fail because of the type checks. If you take a look to [this previous commit ](https://github.com/safe-global/safe-client-gateway-nest/pull/522/commits/7c457dd5db7f8f44b825cc9ec4c8ddf84ca8e889) you'll see I initially tried to solve these errors, blaming a `yarn workspaces focus --production` for installing a different typescript version or so. But the real root cause for those compilation errors was the compiler couldn't infer some variables were non-null after calling lodash's `isArray`.

So the final approach I took is to maintain `@types/lodash` as a production dependency, but I'm not against an alternative approach like removing these lodash-driven type guards and using vanilla TS for it. That would make us able to move this dependency to `devDependencies` again.

In any case, I'd do any follow-up work regarding this in a separate PR 🙂